docs/src/_getting-started/meltano-at-a-glance.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Meltano at a Glance
+description: If you want to know why you should use Meltano, here's a quick overview of the basics of all of it.
+layout: doc
+weight: 1
+---
+
+This document is intended to give you enough technical understanding of Meltano to becoming excited about it and wanting to use it! It's not going to teach you how to use it, we got Tutorials & How Tos for that. When you're ready to start your first Meltano project, we recommend you dive right into our Tutorial.
+
+### What users say
+
+_"For us it's a better day at work when we can use Meltano."_ - Nino Müller, Head of Technology at Substring
+
+
+_"I love Meltano because it’s so pleasant to use with its DevOps and Everything-as-Code style. It is easy to set up, flexible, and integrates with pretty much any orchestrator as well as the data built tool (dbt)"._  - Martin Morset
+
+## Meltano at a glance
+Welcome to your Open Source DataOps Infrastructure to manage and centralize all the data tools in your stack and turn them into your ideal end-to-end data platform. With Meltano, you'll be able to collaboratively build and improve your data platform like a software project; spinning up a service or tool (Airflow, dbt, Great Expectations, Snowflake, etc) and easily configuring, deploying and managing it through a single control plane.
+
+
+### Why companies love to build with Meltano
+
+- **No lock-in**: It's open source and has a strong community, you'll always be free.. (derisk)
+- **It's extensible from day 1**: It's super easy to add a custom connection using the SDKs/EDKs.
```

Comment:
How much do you want to link out to other properties from here? (thinking EDK / SDK links)

Reply:
My feeling tells me, not at all. "At a Glance" is supposed to sell people on Meltano, explain the simplicity, scope and thus tell a potential user why this is of value to him.

 So I don't really want them to go anywhere else if it doesn't help to achieve this goal-